File: SYBCA Sem 4/OOPs & C++/CPP Assignment 1/A1SetB1.cpp
#include <iostream>
using namespace std;

int main() {
    int num;
    cout << "Enter a number to check the factor of it: ";
    cin >> num;

    cout << "Factors of " << num << " are: "; 

    for(int i = 1;i <= num; i++)
    {
        if(num % i == 0)
        {
            cout << i << " ";
        }
    }

    return 0;
}

File: TYBCA/TYBCA Sem 6/GO/Assignment 1/A1SetB1.go
package main
import "fmt"

func main() {
    var num int
    
  fmt.Print("Enter number: ")
  fmt.Scan(&num)

  for i:=1; i < 11; i++ {
  	fmt.Println(num, "x", i, "=", (num*i))
  }
}
